web/src/lib/components/shared-components/upload-cover.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+	import { fade } from 'svelte/transition';
+
+	export let dropHandler: (event: DragEvent) => void;
+	export let dragOverHandler: (event: DragEvent) => void;
+	export let dragLeaveHandler: () => void;
+</script>
+
+<div
+	in:fade={{ duration: 250 }}
+	out:fade={{ duration: 250 }}
+	on:drop={dropHandler}
+	on:dragover={dragOverHandler}
+	on:dragleave={dragLeaveHandler}
+	class="fixed inset-0 w-full h-full z-[1000] flex flex-col items-center justify-center bg-immich-bg dark:bg-immich-dark-bg text-immich-dark-gray dark:text-immich-gray"
```

Comment:
Can we use `bg-gray-100/90 dark:bg-immich-dark-bg/90` for the background? Thank you

Reply:
Changed. I thought about adding some opacity to the overlay but was not sure. Thanks for your opinion!